static/lang/re-en.json

```diff
@@ -2752,6 +2752,14 @@
                 },
                 "MysteryConduit": {
                     "Note": "When you cast a spell of 5th rank or lower that has no duration, instead of spending a spell slot, you can advance the effects of your oracular curse in the same manner as when you cast a revelation spell. A spell cast in this way gains the cursebound trait."
+                },
+                "CurseSeverity": {
+                    "NoneToMinor": "Your curse progresses to its minor stage the first time you finish casting a revelation spell after your daily preparations.",
+                    "MinorToModerate": "If you cast a revelation spell while your minor curse is in effect, you progress to the moderate curse effect immediately after you finish Casting the Spell.",
+                    "MinorToOffGuard": "You become Off-Guard in addition to the effects of the minor curse.",
```

Comment:
```suggestion
                    "MinorToOffGuard": "You become @UUID[Compendium.pf2e.conditionitems.Item.AJh5ex99aV6VTggg]{Off-Guard} in addition to the effects of the minor curse.",
```

Reply:
I ended up not linking this (and doomed) since the curse applies them automatically in both cases. I'll link them anyway though. I think it's good to have a clickable reference.